### PR TITLE
Use Directory.Build.props/targets

### PR DESCRIPTION
**Pull request by natemcmaster**
Reduces duplication across csproj's and makes it easier to apply the same settings to groups of projects.

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <Project>
```
Comment (NTaylorMullen) [inline Directory.Build.props]: Will this file impact our `tooling\Microsoft.VisualStudio.RazorExtension\Microsoft.VisualStudio.RazorExtension.csproj`?

**Comment by NTaylorMullen** [comment]
btw when you add me just as a reviewer it doesn't ping me in my notifications 😢

File: tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj [hunk 2]
```diff
@@ -20,6 +20,8 @@
     <ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot>
     <ExtensionInstallationFolder>Microsoft\RazorLanguageServices</ExtensionInstallationFolder>
     <IsProductComponent>true</IsProductComponent>
+    <ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>
+    <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
```
Comment (natemcmaster) [inline tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj]: This suppresses Directory.Build.* imports.

**Comment by natemcmaster** [comment]
@NTaylorMullen hmm, Ryan B had the same issue. I'll cc you from now on.

**Pull request merged**